uiplib/gui/mainGui.py

```diff
@@ -94,6 +96,7 @@ def flush(self):
                                 icon='warning')
         if ask == 'yes':
             flush_wallpapers(self.settings['pics-folder'])
+            self.update_pics_from_fav_pics()
```

Comment:
My idea behind putting this here is that ,when we click flush then all the images are removed from the pics folder including the user's fav pics . So calling this function here would again place the fav pics of users back to pics folder so that user's can use them to set_wallpapers and other things.